service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1086,13 +722,6 @@ func (m *LatestClientVersionReply) String() string            { return proto.Com
 func (*LatestClientVersionReply) ProtoMessage()               {}
```

Comment:
exported method LatestClientVersionReply.Descriptor should have comment or be unexported